### PR TITLE
Add the query parameter field

### PR DESCRIPTION
**Pull request by LingMQ**
Grab the data from elasticsearch based on the parameter(s). So the
analytics scripts will run against a specific job instead of date
range.

The jira ticket: https://projects.engineering.redhat.com/browse/DATAHUB-253

### CODE REVIEW

**Comment by ghost** [comment]
Looks good to me.  So Ansible will be used to kick off the job?

File: logstash-configs/conf.d/00-input-elasticsearch-1w-param.conf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+input {
```
Comment (t0ffel) [inline logstash-configs/conf.d/00-input-elasticsearch-1w-param.conf:1]: please create a separate directory with configs for this job

File: logstash-configs/conf.d/00-input-elasticsearch-1w-param.conf [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  #Connect to ElasticSearch stage and query for last hour of data from jenkins data.
+  elasticsearch {
+    hosts => ["{{ production_elasticsearch }}"]
+    index => "rhci-logs-*"
+    codec => json_lines
+    query => '{ "query": { "bool": { "must": [ { "term": { "{{ query_parameter_field }}": "{{ query_parameter_value }}" } }, { "range": { "@timestamp": { "gte": "now-7d/d", "lt": "now/d" } } } ] } } }'
```
Comment (t0ffel) [inline logstash-configs/conf.d/00-input-elasticsearch-1w-param.conf:7]: time period must be parameterized

File: ansible/roles/logstash-job-openshift/defaults/main.yaml [hunk 3]
```diff
@@ -14,3 +14,7 @@ job_name: logstash-job-1
 configs_dir: "{{ role_path }}/files/"
 # Define certs_dir to add certificates to the job
 # certs_dir: "{{ role_path }}/files/"
+
+
+query_parameter_field: ci_job.name
```
Comment (t0ffel) [inline ansible/roles/logstash-job-openshift/defaults/main.yaml]: please remove this parameter and hard code ci_job.name field

File: ansible/roles/logstash-job-openshift/defaults/main.yaml [hunk 4]
```diff
@@ -14,3 +14,7 @@ job_name: logstash-job-1
...
+query_parameter_value: Interop-OSP_10-OCP_3.4-MLibVirt-HA-smoke-70e5f-stable-runtest
```
Comment (t0ffel) [inline ansible/roles/logstash-job-openshift/defaults/main.yaml]: rename parameter to reference job name

File: logstash-configs/conf.d/00-input-elasticsearch-1w-param.conf (hunk 1 above)
Comment (LingMQ) [reply logstash-configs/conf.d/00-input-elasticsearch-1w-param.conf:1]: done

File: logstash-configs/conf.d/00-input-elasticsearch-1w-param.conf (hunk 2 above)
Comment (LingMQ) [reply logstash-configs/conf.d/00-input-elasticsearch-1w-param.conf:7]: done

File: ansible/roles/logstash-job-openshift/defaults/main.yaml (hunk 4 above)
Comment (LingMQ) [reply ansible/roles/logstash-job-openshift/defaults/main.yaml]: done

File: ansible/roles/logstash-job-openshift/defaults/main.yaml (hunk 3 above)
Comment (LingMQ) [reply ansible/roles/logstash-job-openshift/defaults/main.yaml]: done

File: logstash-configs/conf.d/00-input-elasticsearch-1w.conf [hunk 5]
```diff
@@ -1,10 +0,0 @@
-input {
```
Comment (t0ffel) [inline logstash-configs/conf.d/00-input-elasticsearch-1w.conf:1]: why are you touching this file?

**Comment by t0ffel** [comment]
Please squash the commits.

**Comment by shgriffi** [comment]
Gryphon99 was me, I responded to this with the wrong account.  I'm switching them now.

File: logstash-configs/conf.d/00-input-elasticsearch-1w.conf (hunk 5 above)
Comment (LingMQ) [reply logstash-configs/conf.d/00-input-elasticsearch-1w.conf:1]: I delete this file cuz the change has been incorporated into file "logstash-configs-prod/conf.d/00-input-elasticsearch.conf"

**Pull request merged**